Data_Structures_Questions.md

```diff
@@ -3,15 +3,15 @@ For each of the methods associated with each data structure, classify it based o
 ## Linked List
```

Comment:
I can see you're really putting in the extra effort with data structures Tim, this is a great start to the Repo. Try and answer everything on this file and study it for tomorrow. I see you have a good grasp of the material even though we only briefly touched on it.